### ISSUE TITLE
twisted.internet.test.test_iocp.SupportTests.test_ipv6AcceptAddress test failure

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8780]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8780|
|Type|defect|
|Created|2016-08-24 07:42:22Z|
|Branch|https://github.com/twisted/twisted/tree/8780-iocp-ipv6|

I noticed in this build log: https://ci.appveyor.com/project/adiroiban/twisted/build/trunk-1012/job/w63y6gjb1lkuk8ar

There was this test failure:

```
Traceback (most recent call last):
  File "c:\projects\twisted\build\py27-coverage-windows\lib\site-packages\twisted\internet\test\test_iocp.py", line 97, in test_ipv6AcceptAddress
    self._acceptAddressTest(AF_INET6, '::1')
  File "c:\projects\twisted\build\py27-coverage-windows\lib\site-packages\twisted\internet\test\test_iocp.py", line 66, in _acceptAddressTest
    SOL_SOCKET, SO_UPDATE_ACCEPT_CONTEXT, pack('P', server.fileno()))
  File "C:\Python27-x64\Lib\socket.py", line 228, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 10057] A request to send or receive data was disallowed because the socket is not connected and (when sending on a datagram socket using a sendto call) no address was supplied
twisted.internet.test.test_iocp.SupportTests.test_ipv6AcceptAddress
-------------------------------------------------------------------------------
```

<details><summary>Searchable metadata</summary>

```
trac-id__8780 8780
type__defect defect
reporter__rodrigc rodrigc
priority__normal normal
milestone__None None
branch__8780_iocp_ipv6 8780-iocp-ipv6
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1472024542935489 1472024542935489
changetime__1474198569965514 1474198569965514
version__None None
owner__Craig_Rodrigues__rodrigc_____ Craig Rodrigues <rodrigc@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

https://github.com/twisted/twisted/pull/521

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc set owner to @rodrigc<br>@rodrigc set status to `closed`|
|-|-|

In changeset b43ebc3999b3351dfb8e33e97371ca40654344c8
```
#!CommitTicketReference repository="" revision="b43ebc3999b3351dfb8e33e97371ca40654344c8"
Merge pull request #521 from twisted/8780-iocp-ipv6

Author: rodrigc
Reviewer: opalmer
Fixes: #8780

Fix AcceptEx() test on Windows
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing the review tag from closed tickets
